bitdefendergravityzone.json

```diff
@@ -0,0 +1,508 @@
+{
+    "appid": "934dfce7-2071-4acb-8402-837c1702f2db",
+    "name": "Bitdefender GravityZone",
+    "description": "This app integrates with Bitdefender GravityZone to execute various containment, corrective and investigative actions",
+    "type": "endpoint",
+    "product_vendor": "Bitdefender",
+    "logo": "bitdefendergravityzone.svg",
+    "logo_dark": "bitdefendergravityzone_dark.svg",
+    "product_name": "GravityZone",
+    "fips_compliant": false,
+    "python_version": "3",
+    "product_version_regex": ".*",
+    "publisher": "Splunk Community",
+    "license": "Copyright (c) 2022 Splunk Inc.",
+    "app_version": "1.0.0",
+    "utctime_updated": "2022-03-22T13:20:25.458448Z",
+    "package_name": "phantom_bitdefendergravityzone",
+    "main_module": "bitdefendergravityzone_connector.py",
+    "min_phantom_version": "5.2.1.78411",
```

Comment:
5.1.0

Reply:
Done in https://github.com/splunk-soar-connectors/bitdefendergravityzone/pull/1/commits/e35b41982cc0e70d9a3d6e85074a2f4ac79cd8df